infer/src/base/Process.mli

```diff
@@ -6,6 +6,7 @@
  *)
 
 open! IStd
+open Core (* exit_or_signal *)
```

Comment:
Likewise here, we could just remove this `open` and be explicit when we use `Core`.